python/dgl/graph.py

```diff
@@ -1244,6 +1272,85 @@ def remove_edges(self, eids):
 
         self._graph = sgi.graph
 
+    @property
+    def parent_nid(self):
+        """Get the parent node ids.
+
+        The returned tensor can be used as a map from the node id
+        in this subgraph to the node id in the parent graph.
+
+        Returns
+        -------
+        Tensor
+            The parent node id array.
+        """
+        return self._parent_nid.tousertensor()
+
+    def _get_parent_eid(self):
+        # The parent eid might be lazily evaluated and thus may not
+        # be an index. Instead, it's a lambda function that returns
+        # an index.
+        if isinstance(self._parent_eid, utils.Index):
+            return self._parent_eid
+        else:
+            return self._parent_eid()
+
+    @property
+    def parent_eid(self):
+        """Get the parent edge ids.
+
+        The returned tensor can be used as a map from the edge id
+        in this subgraph to the edge id in the parent graph.
+
+        Returns
+        -------
+        Tensor
+            The parent edge id array.
+        """
+        return self._get_parent_eid().tousertensor()
```

Comment:
I don't see the value of `parent_nid` and `parent_eid`. They will be replaced by:
```python
g.ndata[dgl.NID]
g.edata[dgl.EID]
```

Reply:
And if a `DGLGraph` will hold a reference to its parent graph, consider add a `parent` property.

Reply:
DONE